challenge_1.exs

```diff
@@ -0,0 +1,83 @@
+defmodule Challenge_1 do
+  @moduledoc """
+
+  Documentación del modulo 'Challenge_1'
+
+  Este modulo da funciones de utilidad, para encontrar la solución al problema
+  'División por 3', este problema consiste en que a partir de una serie formada por
+  un número n, contar la cantidad que hay desde un punto a a un punto b que son divisibles por 3.
+
+  Serie:
+    Si de entrada es 1 se obtiene = 1
+    Si de entrada es 2 se obtiene = 12
+    Si de entrada es 3 se obtiene = 123
+    Si de entrada es n se obtiene = 123...n
+
+  """
+
+  @doc """
```

Comment:
You can add specs here and also make smaller docs. Please do this for all methods.